### PR TITLE
(DOCSP-25113): Document Metrics endpoint

### PR DESCRIPTION
**Pull request by cbush**
- Adds new endpoint

### Jira

- https://jira.mongodb.org/browse/DOCSP-25113

### Staged Changes (Requires MongoDB Corp SSO)

- https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/metrics-api/admin/api/v3/#tag/metrics

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/openapi-admin-v3.yaml [hunk 1]
```diff
@@ -1989,6 +1989,41 @@ paths:
       responses:
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/metrics":
+    get:
+      tags:
+        - metrics
```
Comment (nlarew) [inline source/openapi-admin-v3.yaml:1995]: Let's also add an entry for the `metrics` tag to `components.tags` near the bottom of the file.

File: source/openapi-admin-v3.yaml [hunk 2]
```diff
@@ -1989,6 +1989,41 @@ paths:
...
+      operationId: adminGetMetrics
+      summary: Retrieve App Services metrics
```
Comment (nlarew) [inline source/openapi-admin-v3.yaml:1997]: Maybe also add a `description` field with more details/links out? e.g. what metrics are available, how long do we hang on to metrics data, etc.

File: source/openapi-admin-v3.yaml [hunk 3]
```diff
@@ -1989,6 +1989,41 @@ paths:
...
+            The ISO 8601 date and time of the end of the query period.
+          schema:
+            type: string
+          required: true
+        - name: granularity
+          in: query
+          description: |-
+            Specifies the granularity of the query period as an ISO-8601
+            duration value. Examples include "P31D" (31 day), "PT1H" (1 hour),
+            and "PT5M" (5 minutes).
```
Comment (nlarew) [inline source/openapi-admin-v3.yaml:2020]: Do we have a list of these granularity strings somewhere? Maybe a link out to other docs or (even better IMO) an `enum` in the schema that lists all the options.

Comment (cbush) [reply source/openapi-admin-v3.yaml:2023]: It's a format actually specified in ISO-8601, so we can't really enumerate all of the options. I gave a few examples here but we could link out to Wikipedia?

**Review by nlarew — APPROVED** [review]
LGTM!

File: source/openapi-admin-v3.yaml [hunk 4]
```diff
@@ -1992,6 +1992,44 @@ paths:
...
+          description: |-
+            The ISO 8601 date and time of the end of the query period.
+          schema:
+            type: string
+          required: true
+        - name: granularity
+          in: query
+          description: |-
+            Specifies the granularity of the query period as an
+            :wikipedia:`ISO-8601 duration value <ISO_8601#Durations>`. Examples
```
Comment (nlarew) [inline source/openapi-admin-v3.yaml:2025]:
```suggestion
            [ISO-8601 duration value](https://en.wikipedia.org/wiki/ISO_8601#Durations). Examples
```

OpenAPI uses Markdown, not rST

**Pull request merged**